docker/docker-py3-kms/Dockerfile

```diff
@@ -52,7 +52,7 @@ COPY docker/start.py /usr/bin/start.py
 RUN chmod 755 /usr/bin/entrypoint.py
 
 WORKDIR /home/py-kms
-#USER py-kms
```

Comment:
@simonmicro why is this `USER` directive commented out? Should we be root-less?

Reply:
The entrypoint is already responsible for dropping the permissions - take a look into it and its `UID` capabilites :grin: